packages/core/src/layout/Sidebar/Bar.tsx

```diff
@@ -23,7 +23,7 @@ import { SidebarPinStateContext } from './Page';
 
 const useStyles = makeStyles<BackstageTheme>(theme => ({
   root: {
-    zIndex: 1000,
+    zIndex: 1401,
```

Comment:
This can be a higher number in future. The Dismissable Banner component inherits the `z-index` of 1400 from the Material UI's [`Snackbar`](https://material-ui.com/components/snackbars/) component.